src/entity/user/UserStreak.ts

```diff
@@ -0,0 +1,23 @@
+import { Index, Column, PrimaryColumn, Entity } from 'typeorm';
+
+@Entity()
+export class UserStreak {
+  @PrimaryColumn({ type: 'text' })
+  @Index()
```

Comment:
PrimaryColumn is already indexed, we don't need that

Reply:
Should be fixed now 🚀